### ISSUE TITLE
Typescript errors and patch-package fixes

### ISSUE THREAD

**Issue by thespacemanatee** [Naeemur/react-native-wheel-color-picker#14]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-wheel-color-picker@1.1.0` for the project I'm working on.

Currently, all props on the ColorPicker component needs to be specified even though in reality it doesn't need to as you have defined static default props in the component itself. onColorChange and onColorChangeComplete also had wrong types. This patch fixes both of the issues that I mentioned, feel free to incorporate them if you agree with the changes.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-wheel-color-picker/types.d.ts b/node_modules/react-native-wheel-color-picker/types.d.ts
index 365aab5..6fec768 100644
--- a/node_modules/react-native-wheel-color-picker/types.d.ts
+++ b/node_modules/react-native-wheel-color-picker/types.d.ts
@@ -1,20 +1,20 @@
 import * as React from 'react';
 
 export interface ColorPickerProps extends React.Props<ColorPicker> {
-  row: boolean,
-  noSnap: boolean,
-  thumbSize: number,
-  sliderSize: number,
-  discrete: boolean,
-  swatches: boolean,
-  swatchesLast: boolean,
-  swatchesOnly: boolean,
-  color: string,
-  shadeWheelThumb: boolean,
-  shadeSliderThumb: boolean,
-  autoResetSlider: boolean,
-  onColorChange(): void,
-  onColorChangeComplete(): void,
+  row?: boolean,
+  noSnap?: boolean,
+  thumbSize?: number,
+  sliderSize?: number,
+  discrete?: boolean,
+  swatches?: boolean,
+  swatchesLast?: boolean,
+  swatchesOnly?: boolean,
+  color?: string,
+  shadeWheelThumb?: boolean,
+  shadeSliderThumb?: boolean,
+  autoResetSlider?: boolean,
+  onColorChange?(color: string): void,
+  onColorChangeComplete?(color: string): void,
 }
 
 declare class ColorPicker extends React.Component<ColorPickerProps, any> {
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by LinusU** [comment]
This was fixed with #8, not released yet though

**Comment by nadilas** [comment]
> This was fixed with #8, not released yet though

could we get a release?

**Comment by Naeemur** [comment]
Version 1.2.0 released with PR merged and issues fixed.
